scandium-core/src/test/java/org/eclipse/californium/scandium/dtls/ServerHandshakerTest.java

```diff
@@ -429,13 +429,12 @@ private void processClientHello(int epoch, long sequenceNo, int messageSeq, byte
 	}
 
 	private static byte[] newHandshakeMessage(HandshakeType type, int messageSeq, byte[] fragment) {
-		int length = 8 + 24 + 16 + 24 + 24 + fragment.length;
```

Comment:
That was a nice spotted bug. Detected by the new introduced check, if handshake message has left unprocessed payload.